README.md

```diff
@@ -491,26 +491,78 @@ If you specify both **alert** and **data**, target device will receive both noti
-* Run: `docker cp mongoose_push:/opt/app/var/sys.config sys.config` on you docker host (this will get the current `sys.config` to your `${CWD}`)
-* Modify the `sys.config` as you see fit (for metrics, see above)
-* Stop MongoosePush docker container and restart it with the modified `sys.config` as volume in `/opt/app/sys.config` (yes, this is not the path we used to copy this file from, this is an override)
+MongoosePush 2.1 provides metrics in the Prometheus format on the `/metrics` endpoint.
+This is a breaking change compared to previous releases.
+Existing dashboards will need to be updated.
 
+It is important to know that metrics are created inside MongoosePush only when a certain event happens.
+This may mean that a freshly started MongoosePush node will not have all the possible metrics available yet.
 
-### Available metrics
+#### Available metrics
 
-The following metrics are available:
+##### Histograms
+
+For more details about the histogram metric type please go to https://prometheus.io/docs/concepts/metric_types/#histogram
+
+###### Notification sent time
+
+`mongoose_push_notification_send_time_microsecond_bucket{error_category=${CATEGORY},error_reason=${REASON},service=${SERVICE},status=${STATUS},le=${LE}}`
+`mongoose_push_notification_send_time_microsecond_sum{error_category=${CATEGORY},error_reason=${REASON},service=${SERVICE},status=${STATUS}}`
+`mongoose_push_notification_send_time_microsecond_count{error_category=${CATEGORY},error_reason=${REASON},service=${SERVICE},status=${STATUS}}`
 
-* `mongoose_push_apns_state_get_default_topic_count`
-* `mongoose_push_notification_send_time_bucket{error_category=${CATEGORY},error_reason=${REASON},service=${SERVICE},status=${STATUS},le=${LENGTH}}`
-* `mongoose_push_notification_send_time_sum{error_category=${CATEGORY},error_reason=${REASON},service=${SERVICE},status=${STATUS}}`
-* `mongoose_push_notification_send_time_count{error_category=${CATEGORY},error_reason=${REASON},service=${SERVICE},status="${STATUS}}`
 Where:
-* **CATEGORY** is an arbitrary error category term or empty string
-* **REASON** is an arbitrary error reason term or empty string
-* **SERVICE** is either `fcm` or `apns`
-* **STATUS** is either `success` or `error`
-* **LENGTH** is either `100` or `250` or `500` or `1000` or `+Inf`
+* `STATUS` is `"success"` for the successful notifications or `"error"` in all other cases
+* `SERVICE` is either `"apns"` or `"fcm"`
+* `CATEGORY` is an arbitrary error category term (in case of `status="error"`) or an empty string (when `status="success"`)
+* `REASON` is an arbitrary error reason term (in case of `status="error"`) or an empty string (when `status="success"`)
+* `LE` defines the `less or equal` values for buckets, currently `1000`, `10_000`, `25_000`, `50_000`, `100_000`, `250_000`, `500_000`, `1000_000` or `+Inf`
+
+> **NOTE**
+> A mesurment of value 50_000 will be added to all buckets which are less or equal to 50_000.
```

Comment:
Not sure if I understand this right, I think this is the other way around. The bucket contains measurements that are less or equal to the bucket size.

`More particularly they're counters which form a cumulative histogram, le stands for less than or equal to. So 26688 requests took less than or equal to 200ms, 27760 requests took less than or equal to 400ms, and there were 28860 requests in total. ` 
[https://www.robustperception.io/how-does-a-prometheus-histogram-work](https://www.robustperception.io/how-does-a-prometheus-histogram-work)

Reply:
Right, I messed with that. Good catch!